suite-native/app/globalPolyfills.js

```diff
@@ -1,10 +1,20 @@
 /**
  * Inject global objects to react-native app
  */
+import '@formatjs/intl-getcanonicallocales/polyfill';
+import '@formatjs/intl-locale/polyfill';
+import '@formatjs/intl-pluralrules/polyfill';
+import '@formatjs/intl-pluralrules/locale-data/en';
+import '@formatjs/intl-numberformat/polyfill';
+import '@formatjs/intl-numberformat/locale-data/en';
+import '@formatjs/intl-datetimeformat/polyfill';
+import '@formatjs/intl-datetimeformat/locale-data/en';
+import '@formatjs/intl-datetimeformat/add-all-tz';
 
 global.Buffer = require('buffer').Buffer;
 global.process = require('process');
-// global.Intl = require('intl');
+global.BigInt = require('big-integer');
```

Comment:
I am using this because it is a part of `wallet-utils` https://github.com/trezor/trezor-suite/blob/feat/suite-common-formatters/suite-common/wallet-utils/src/localizeNumber.ts#L23 used both by suite and suite native... not sure if we can use this react-native lib.

Reply:
Lib will be replaced under the hood by Metro bundler, but we can create follow-up